GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2_truss.cfg

```diff
@@ -0,0 +1,207 @@
+	@maxTemp = 1073.15
+
+	// Area gives each module 30 kW at tech level 6
+	SolarConfig
+	{
+		area = 113.21
+		level = 6
+	}
+
+	// Disable SolarConfig from changing the dummy module for solar tracking
+	@MODULE[ModuleDeployableSolarPanel]:HAS[#pivotName[SARJ]]
+	{
+		@chargeRate = 0.000001
+		realismOverhaulSolarIgnore = true
+	}
+
+	@MODULE[ModuleB9PartSwitch]:HAS[#moduleID[topMount]]
+	{
+		// Average of S5/P5 truss
+		@SUBTYPE[S5]
+		{
+			%addedMass = 1.841
+		}
+	}
+
+	@MODULE[ModuleB9PartSwitch]:HAS[#moduleID[bottomMount]]
+	{
+		@SUBTYPE[SARJ]
+		{
+			%addedMass = 1.134
+		}
+	}
+}
+
+//	ISS S0 truss
+//	Diameter: 4.6m
+//	Length: 13.4m
+//	Mass: 13,970kg (30,800lb)
+//	Source: http://spacecraftearth.com/#smoothscroll-portfolio-module
+@PART[ht2_truss_S0]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.81
+	@mass = 13.970
+	@title = ISS S0 Truss Segment
+	@manufacturer = Boeing
+	@description = The S0 truss, (also called the Center Integrated Truss Assembly Starboard 0 Truss) forms the center backbone of the Space Station. It was attached on the top of the Destiny Laboratory Module during STS-110 in April 2002. S0 is used to route power to the pressurized station modules and conduct heat away from the modules to the S1 and P1 Trusses.
+	@maxTemp = 1073.15
+
+	@RESOURCE[ElectricCharge]
+	{
+		@name = ElectricCharge
+		%amount = 3000
+		%maxAmount = 3000
+	}
+}
+
+//	ISS S1/P1 truss
+//	Diameter: 4.6m
+//	Length: 13.7m
+//	Mass: 9,124kg // 12,524kg (27,611lb) inc triple HRS (seperate part)
+//	Source: https://space.skyrocket.de/doc_sdat/its-s1.htm
+@PART[ht2_truss_S1]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.81
+	@mass = 9.124
+	@title = ISS S1/P1 Truss Segment
+	@manufacturer = Boeing
+	@description = The P1 and S1 trusses (also called the Port and Starboard Side Thermal Radiator Trusses) are attached to the S0 truss, and contain carts to transport the Canadarm2 and astronauts to worksites along the space station. They each flow 290 kg (637 lb) of anhydrous ammonia through three heat rejection radiators. The S1 truss was launched on STS-112 in October 2002 and the P1 truss was launched on STS-113 in November 2002.
+	@maxTemp = 1073.15
+
+	@RESOURCE[ElectricCharge]
+	{
+		@name = ElectricCharge
+		%amount = 2500
+		%maxAmount = 2500
+	}
+
+	MODULE
+	{
+		name = ModuleRealAntenna
+		antennaDiameter = 1.0
+	}
```

Comment:
Why does the truss part have a dedicated antenna?  Is it modelled?
This should be guarded with a NEEDS, at least, since RO does not require RA.
```%MODULE[ModuleRealAntenna]:NEEDS[RealAntennas] { %antennaDiameter = 1.0 }```

Reply:
Fairly sure the antenna is not modelled, but was added here because the real truss has an antenna.

Should it be present in RO?

Reply:
I would typically think trusses are structures that you add parts like antennas to.  If it also isn't modelled, I'd stick more closely to that view.

Reply:
Removed the antenna